firebase-perf/src/main/java/com/google/firebase/perf/FirebasePerfRegistrar.java

```diff
@@ -31,8 +31,6 @@
 import com.google.firebase.platforminfo.LibraryVersionComponent;
 import com.google.firebase.remoteconfig.RemoteConfigComponent;
 import com.google.firebase.sessions.FirebaseSessions;
```

Comment:
Should we remove all references to `FirebaseSessions`?

Reply:
This is all cleaned up now after taking in the latest change of sessions-nine branch.